lnrpc/walletrpc/walletkit.proto

```diff
@@ -538,6 +543,11 @@ message ListAddressesResponse {
     repeated AccountWithAddresses account_with_addresses = 1;
 }
 
+message GetTransactionRequest {
+    // Transaction hash.
+    string tx_hash = 1;
```

Comment:
Since technically this is the TXID (which is the by-reversed hash of the transaction), this should either be called `txid` if it's a string, or be turned into a `bytes` field which then expects the non-reversed bytes of the transaction hash.

Reply:
I'd vote for renaming to `txid` and leaving the `string` type, since that makes the REST call easier as well, since `bytes` are a bit harder to handle.

Reply:
Done